src/public/modules/forms/base/components/start-page.client.component.js

```diff
@@ -75,6 +75,11 @@ function startPageController(SpcpSession, Toastr) {
       return
     }
 
+    const banner = document.getElementById('notification-banner')
+    const bannerHeight = banner ? `${banner.offsetHeight + 20}px` : '0px'
+    // Update height of header
+    header.style.paddingTop = bannerHeight
```

Comment:
not a fan of this haha, I don't think we should mix vanilla JS with angularJS, and this positioning logic should be the job of the parent. but letting it go this time haha